www/restinio/Portfile

```diff
@@ -0,0 +1,65 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+PortSystem                  1.0
+PortGroup                   github 1.0
+PortGroup                   cmake 1.1
+PortGroup                   compiler_blacklist_versions 1.0
+
+github.setup                Stiffstream restinio 0.4.5 v.
+categories                  www devel
+platforms                   darwin
+license                     BSD
+maintainers                 @Lord-Kamina
+description                 Header-only C++14 library that gives you an embedded HTTP/Websocket server.
+depends_build-append        port:boost port:openssl port:pcre port:zlib bin:doxygen:doxygen
+depends_lib-append          port:asio \
+                            port:http-parser \
+                            path:include/fmt/format.h:libfmt \
+                            path:include/fmt/ostream.h:libfmt \
+                            path:include/fmt/time.h:libfmt
```

Comment:
These 3 path dependencies should be changed into a single port dependency on libfmt.

Reply:
@mf2k I made the changes (haven't commited yet) but when trying it out with port -vst, trace mode hides the dependency if I declare it as 
```depends_build port:autoconf```
Any idea what could be going wrong?

Reply:
@Lord-Kamina: Which port are you referring to? If it is asio, then it could be a weird issue with use_autoconf. It is not documented and I don't understand how it works. Maybe it automatically adds the build dependency and you are overriding it. Maybe try using "-append" or just check if it gets added automatically in the first place?